### ISSUE TITLE
If I want to detect one class, should I set use_sigmoid = True?

### ISSUE THREAD

**Issue by guiyuliu** [open-mmlab/mmdetection#3748]
I only want to detect one class, e.g car ,My dataset annotation just has one class label .
should I set set use_sigmoid = True?
I see the source code about this config parameter is 
```
        if self.use_sigmoid_cls:
            self.cls_out_channels = self.num_classes
        else:
            self.cls_out_channels = self.num_classes + 1
```
if I set use_sigmoid = False, the loss function will be binary loss ?
if  I set use_sigmoid = True, what will be the loss function
please?

**Comment by yuzhj** [comment]
set num_classes=1

**Comment by ZwwWayne** [comment]
No you do not need to do that. You just need to set `num_classes=1`.
If you set use_sigmoid = False, the loss function will be still be cross_entropy loss, though the number of classes is 2, 1 foreground and 1 background.
If you set use_sigmoid = True, it will be binary cross_entropy loss. Because now it only have one logits.

**Comment by ZwwWayne** [comment]
The question seems to be answered thus the issue is closed. Feel free to reopen it if you have any further questions.
